stephanie's onboarding

cauliiflower opened the pull request:
# Strategy Onboarding

## TODO List

Onboarding tasks:

- [x ] Task 1: Implement `fibonacci_term()` in `fibonacci.py`
  - All tests in `test_fibonacci.py` should pass, run `pytest` command to check
- [x ] Task 2: Implement `find_min_index()` and `reverse_str_arr()`
  - Write tests that cover the logic that you wrote in `test_array.py`
- [x ] Task 3: Process and visualize sample data in `matplot.py`
  - Read data in CSV using `pandas` or `csv`, handle the missing values, plot using `matplotlib`
  - Include a screenshot of your graph in the PR

Some extra items:

- [ ] Completed [strategy welcome survey](https://forms.gle/SKozF7TUn8o385zd8)

<!-- 
## Questions

Feel free to put any questions, comments, or concerns here. If you have anything you want reviewers to double-check, also good to flag it here!
-->

probro27 commented on onboarding/fibonacci.py:
```diff
@@ -14,5 +14,17 @@ def fibonacci_term(n):
-    pass
+    # checking valid inputs
+    if type(n) != int or n < 1:
+      return None
```
raise TypeError if type is not int. Please check the test cases and do appropriately.

probro27 reviewed (changes requested):
@cauliiflower Please check the review I have given. Also please attach the images produced by the matplotlib program. Thanks :)

cauliiflower commented:
@probro27 Hi, this is a kinda stupid question, but I've updated my files and committed them onto my forked branch for my onboarding pr. How do I updated the files here? Thanks :))

cauliiflower commented:
> @probro27 Hi, this is a kinda stupid question, but I've updated my files and committed them onto my forked branch/repository for my onboarding pr. How do I updated the files here? Thanks :))

Oops I think I committed to a new repository rather than my old one. How would I merge the two together or merge my new one with this pull request? Thanks.

Also, I didn't include graph 4 because it made my computer lag a bit when trying to process it.

probro27 commented:
@cauliiflower That's fine. Graph 4 is a bit weird. So I would suggest make the same changes in the prev repo since you can't really merge two repositories. 
If you are not able to do so, I can help you out over a call.

probro27 reviewed (approved):
@cauliiflower LGTM! Thanks for the PR!
We'll reach out to you for a one on one and your first ticket.
Welcome to the team!
